hack/update-version.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+OPERATOR_VERSION="0.0.23"
+MANIFESTS_DIR="manifests"
+OPERATOR_VERSION_UPDATE=$1
+
+if [ "${OPERATOR_VERSION_UPDATE}" != "${OPERATOR_VERSION}" ] ; then
+    NEW_VERSION="$(echo ${OPERATOR_VERSION_UPDATE} | tr '.' '\n' | tail -1)"
+    OLD_VERSION="$(echo ${OPERATOR_VERSION} | tr '.' '\n' | tail -1)"
+    EXPECTED_VERSION="$(echo ${OLD_VERSION} + 1 | bc )"
+    if [ "${NEW_VERSION}" ==  "${EXPECTED_VERSION}" ] ; then
+        sed -i -e 's/0.0.'${OLD_VERSION}'/0.0.'${NEW_VERSION}'/g' Makefile
```

Comment:
You can export this `sed` action to a `function funcion_name() {}`, and call out this function as you need.